pkg/seven/state/builder.go

```diff
@@ -227,64 +221,18 @@ func SolverSingleUpstream(u *v1.Upstream, swg ServiceWorkerGroup, wg *sync.WaitG
 						needToUpdate = false
 					}
 				}
-				if needToUpdate {
-					// 1.sync memDB
-					upstreamDB := &db.UpstreamDB{Upstreams: []*v1.Upstream{u}}
-					if err := upstreamDB.UpdateUpstreams(); err != nil {
-						log.Errorf("solver upstream failed, update upstream to local db failed, err: %s", err.Error())
-						errNotify = err
-						return
-					}
-
-					// 2.sync apisix
-					var cluster string
-					if u.Group != nil {
-						cluster = *u.Group
-					}
+				if needToUpdate || (u.FromKind != nil && *u.FromKind == WatchFromKind) {
```

Comment:
Need to differentiate `service registration` and `CRD synchronization`, 
When `FromKind == WatchFromKind`, the upstream info only contains nodes, and will lose other information in upstream, such as loadBalance infomation.

Reply:
Fixed.